### DISCUSSION TITLE
lolMiner 1.75

### DISCUSSION THREAD

**Discussion by Lolliedieb** [Lolliedieb/lolMiner-releases#1931] [General]
### Changes
- Improved Ironfish mining speed by 4-5% on all supported GPUs.
- Added support to mine Alephium on AMD GCN3 (RX 400, Rx500) and newer GPUs
- Added support to mine Ironfish on AMD GCN3 gpus
- Renamed Blake3-Ironfish to Blake3-Iron to make it identical to competing miner softwares and better detectable by some mining OS

### Fixes
- Fixed a bug causing to little shares submitted when mining Alephium on Nvidia GPUs
- Fixed a bug causing high stale and defect share counts when mining Ironfish on AMD GPUs
- Fixed a bug causing AMD RX 7900 series GPUs not starting up Ironfish mining 
- Fixed a bug causing only defect shares on Ironfish mining when a pool did not zero the Grafitti part of the block header (e.g. ezil pool)

### mmpOS
Open console and write in it:
- `wget https://github.com/Lolliedieb/lolMiner-releases/releases/download/1.75/lolMiner_v1.75_Lin64.tar.gz && tar -xvf lolMiner_v1.75_Lin64.tar.gz && agent-stop && sudo cp -adpR 1.75/lolMiner /opt/mmp/miners/lolminer-equihash && agent-start`

###  RaveOS 
Open console and write in it:
- `stop && wget https://github.com/Lolliedieb/lolMiner-releases/releases/download/1.75/lolMiner_v1.75_Lin64.tar.gz && tar -xvf lolMiner_v1.75_Lin64.tar.gz && cp -adpR 1.75/lolMiner /app-data/miners/lolminer-1.74 &&  start`

###  HiveOS
Open console and write in it:
- `wget https://github.com/Lolliedieb/lolMiner-releases/releases/download/1.75/lolMiner_v1.75_Lin64.tar.gz && tar -xvf lolMiner_v1.75_Lin64.tar.gz && miner stop && cp -adpR 1.75/lolMiner $(ls -d1 /hive/miners/lolminer/*/ | tail -1) && miner start`

###  MinerstatOS
Open console and write in it:
- `mstop; cd /tmp; wget https://github.com/Lolliedieb/lolMiner-releases/releases/download/1.75/lolMiner_v1.75_Lin64.tar.gz; tar -xvf lolMiner_v1.75_Lin64.tar.gz; sudo cp -adpR 1.75/lolMiner /home/minerstat/minerstat-os/clients/lolminer; mstart`

<hr /><em>This discussion was created from the release <a href='https://github.com/Lolliedieb/lolMiner-releases/releases/tag/1.75'>lolMiner 1.75</a>.</em>

**Comment by qingyan888** [comment]
When will LOLminer support sha512256D algorithm？

**Reply by Sun1-nxt** [reply]
Switching to IRON does not work in version 1.75. The calculator shows a profit of 0 for any hashrate values. Other algorithms switches and shows the profit correctly.
